xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.cpp

```diff
@@ -40,17 +40,29 @@ using namespace DXVA;
 using namespace Microsoft::WRL;
 using namespace std::chrono_literals;
 
+// clang-format off
 DEFINE_GUID(DXVADDI_Intel_ModeH264_A,      0x604F8E64,0x4951,0x4c54,0x88,0xFE,0xAB,0xD2,0x5C,0x15,0xB3,0xD6);
 DEFINE_GUID(DXVADDI_Intel_ModeH264_C,      0x604F8E66,0x4951,0x4c54,0x88,0xFE,0xAB,0xD2,0x5C,0x15,0xB3,0xD6);
 DEFINE_GUID(DXVADDI_Intel_ModeH264_E,      0x604F8E68,0x4951,0x4c54,0x88,0xFE,0xAB,0xD2,0x5C,0x15,0xB3,0xD6);
 DEFINE_GUID(DXVADDI_Intel_ModeVC1_E,       0xBCC5DB6D,0xA2B6,0x4AF0,0xAC,0xE4,0xAD,0xB1,0xF7,0x87,0xBC,0x89);
 DEFINE_GUID(DXVA_ModeH264_VLD_NoFGT_Flash, 0x4245F676,0x2BBC,0x4166,0xa0,0xBB,0x54,0xE7,0xB8,0x49,0xC3,0x80);
 DEFINE_GUID(DXVA_Intel_VC1_ClearVideo_2,   0xE07EC519,0xE651,0x4CD6,0xAC,0x84,0x13,0x70,0xCC,0xEE,0xC8,0x51);
 
+#ifndef D3D11_DECODER_PROFILE_AV1_VLD_PROFILE0
+DEFINE_GUID(D3D11_DECODER_PROFILE_AV1_VLD_PROFILE0, 0xb8be4ccb,0xcf53,0x46ba,0x8d,0x59,0xd6,0xb8,0xa6,0xda,0x5d,0x2a);
```

Comment:
Are these defines added in a specific version? What is the minimum windows version needed to support this?

Reply:
This is not need to build Windows x64 using VS 2022 and latest Windows 11 SDK but is added mainly to able to build UWP as we are using Windows 10 SDK (10.0.18362.0) and not includes these GUIDS. Also allows users to continue build x64 with older Windows SDK's and Visual Studio 2019.

It can be removed in the future by increasing minimal Windows SDK version for UWP (although it was not trivial to do it with VS 2019, maybe now with VS 2022 is more easy).

Reply:
Can we gate based on SDK define instead of #if undef?